src/components/ThemesSection.tsx

```diff
@@ -137,8 +137,12 @@ const ThemesSection = ({
                 width="48"
                 height="48"
               />
-              <h3 className="font-semibold text-lg sm:mt-1">{theme.name}</h3>
-              <div className="leading-4 max-w-[8.5rem]">{theme.desc}</div>
+              <h3 className="font-medium font-semibold text-lg sm:mt-1">
```

Comment:
it has to be either medium or semibold

Reply:
I removed all the bold ones. It looks better now.